notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<p>Should we add a little bit more about this target? I know it's artificial that we choose our targets to display based on a post-hoc view of what we find, but we could motivate it somewhat.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I think we also talked about using the example in NB1, since that contains "dips" in the lightcurves every 2.5 days, which are actually caused by changes in the reaction wheel speeds and loss of fine pointing.</p><p> </p><p>This should apply to any target from Sector 1 that we process manually, as lightkurve will exclude these "desaturation" events by default</p>

Reply:
<p>I tried to use the exmaple form NB1 but it did not produce the result we thought it would. When I removed the bad data nothing changed about the curve. I can show you to confirm this when we meet. </p>

Reply:
<p>If we do go this route, I think it would be ok to justify this by saying something vague about the fact that we're looking at a lightcurve that is tricky to analyze</p>